tests/acceptance/test_sro_measures_dsl.py

```diff
@@ -0,0 +1,157 @@
+from databuilder import Measure
+from databuilder.tables import clinical_events, practice_registrations
+
+from .codelists import (
+    alt_codelist,
+    asthma_codelist,
+    cholesterol_codelist,
+    copd_codelist,
+    hba1c_codelist,
+    medication_review_codelist,
+    qrisk_codelist,
+    rbc_codelist,
+    sodium_codelist,
+    systolic_bp_codelist,
+    tsh_codelist,
+)
+
+
+def test_sro_measures_study(cohort, subtest):
+    index_date = "2020-11-01"
+    end_of_index_month = "2020-11-30"
+
+    measures_codelists = {
+        "medication_review": medication_review_codelist,
+        "systolic_bp": systolic_bp_codelist,
+        "qrisk": qrisk_codelist,
+        "cholesterol": cholesterol_codelist,
+        "alt": alt_codelist,
+        "tsh": tsh_codelist,
+        "rbc": rbc_codelist,
+        "hba1c": hba1c_codelist,
+        "sodium": sodium_codelist,
+        "asthma": asthma_codelist,
+        "copd": copd_codelist,
+    }
+
+    with subtest("population"):
+        filtered_practice_registrations = (
+            practice_registrations.filter(
+                practice_registrations.date_start <= index_date
+            )
+            # TODO: Reinstate the following when Predicates support logic operations
+            # .filter(
+            #     (practice_registrations.date_end >= index_date)
+            #     | (practice_registrations.date_end != None)
+            # )
+            .filter(practice_registrations.date_end >= index_date)
+        )
+
+        is_registered = filtered_practice_registrations.exists_for_patient()
+
+        # TODO: The following would be nice
+        # not_dead = patient_demographics.date_of_death == None
```

Comment:
This isn't supported, although we've discussed it.

Reply:
(I mean `patient_demographics.date_of_death` as sugar for `atient_demographics.select_column(patient_demographics.date_of_death)`.